### ISSUE TITLE
Cropping not working correctly

### ISSUE THREAD

**Issue by Vandersteen** [xamarin/SignaturePad#34]
Hello everyone,

Am I making a mistake or is the cropping not working correctly in IOS?
(Extra space at the bottom and right)

When I use

```
GetImage(UIColor.Black, UIColor.Gray, true, true)
```

I get a result like: 
![download](https://cloud.githubusercontent.com/assets/9284334/16077542/d17e3912-32f9-11e6-8c42-b33d83955d7a.png)

Using

```
GetImage(UIColor.Black, UIColor.Gray, true, false)
```

Gives me the following result:
![download 1](https://cloud.githubusercontent.com/assets/9284334/16077567/f448b134-32f9-11e6-9070-f0e72fdc145f.png)

However in Android:

Using:

```
GetImage(Android.Graphics.Color.Black, Android.Graphics.Color.Gray, true, true
```

Result: 
![download 2](https://cloud.githubusercontent.com/assets/9284334/16077884/c4973de6-32fb-11e6-93f2-bd37ee1297b4.png)
)

Using:

```
GetImage(Android.Graphics.Color.Black, Android.Graphics.Color.Gray, true, false)
```

Result: 
![download 3](https://cloud.githubusercontent.com/assets/9284334/16077890/d370514a-32fb-11e6-8e36-bc43a5f56633.png)

PS: I'm saving the image as png for transparency, I added the background to demonstrate the extra space

**Comment by mattleibow** [comment]
This should be fixed in v2.0.

**Comment by gtbuchanan** [comment]
Same occurs on Xamarin.Forms with iOS. iPad is the worst since the screen is larger. Thanks for taking a look.

**Comment by mattleibow** [comment]
Fixed in v2.0.
